Apps/W1/Shopify/app/src/Payments/Codeunits/ShpfyPaymentTermAPI.Codeunit.al

```diff
@@ -0,0 +1,59 @@
+namespace Microsoft.Integration.Shopify;
+
+/// <summary>
+/// Codeunit Shpfy Payment Terms API (ID 30168).
+/// </summary>
+codeunit 30168 "Shpfy Payment Terms API"
+{
+    var
+        JsonHelper: Codeunit "Shpfy Json Helper";
+
+    /// <summary>
+    /// Synchronizes payment terms from shopify, ensuring that the payment terms are up-to-date with those defined in the shopify store.
+    /// </summary>
+    /// <param name="ShopCode">Shopify shop code to be used.</param>
+    internal procedure PullPaymentTermsCodes(ShopCode: Code[20])
```

Comment:
nit: split the function. Let the record creation be in a separate function. Separate from the communication